reference/viewpoint.md

```diff
@@ -43,14 +43,36 @@ The `near`, `far` and the `fieldOfView` fields define together the viewing frust
 3D shape outside this frustum won't be rendered. Hence, shapes too close
 (standing between the camera and the near plane) won't appear.
 
-The `follow` field can be used to specify the name of a robot (or other object)
-that the viewpoint will follow in translation (traveling movement) during the simulation. If the string is
-empty, or if it does not correspond to any object, then the viewpoint will
-remain fixed. The `follow` field is automatically updated when setting the solid
-to be followed from the `View / Follow Object` menu item. If multiple solid
-instances with the same name exist, the instance to be followed is identified by
-adding the instance number to the `follow` field value using the format
-"`<name>:<number>`".
+The `follow` field can be used to specify the name of a robot (or other solid object)
+that the viewpoint will follow in translation (traveling movement) during the simulation.
+If the string is empty, or if it does not correspond to any solid object, then the viewpoint will remain fixed.
+The `follow` field is automatically updated when setting the solid to be followed from the `View / Follow Object` menu item.
+Given that the same [Solid](solid.md).name field value could be used by different [Solid](solid.md) nodes,
+the `follow` field value also contains the name of the ancestor [Solid](solid.md) nodes.
+The list of [Solid](solid.md) names is joined using the character ':' and the '\\' and ':' characters contained in a [Solid](solid.md) name is escaped by prepending '\\'.
+For example, please consider the following structure where the `TARGET` solid is followed:
+```
+Robot {
+  Solid {
+    children [
+      Solid {
+        name "solid"
+      }
+    ]
+    name "first solid"
+  }
+  Solid {
+    children {
+      DEF TARGET Solid {
+        name "target\solid" // <- followed object
+      }
+    }
+    name "second:solid"
+  }
+  name "robot"
+}
+```
+Then, `follow` value will be "robot:second\:target\\solid".
```

Comment:
That seems wrong, shouldn't it be `"robot:second\:solid:target\\solid"` instead?

Reply:
Yes, thank you for pointing it out.